Update Data Life Cycle - Sharing (#283)
elixir-europe/rdm-toolkit: pages/rdm_cycle/sharing.md

---
title: Sharing
contributors: [Flora D'Anna, Bert Droesbeke, Niclas Jareborg]
keywords:
---

## What is Data Sharing?

> Data Sharing means making datasets known and available to the global research community and to society at large. Data sharing can be done at any time during the research data life cycle, but at the latest, data should be made available at the time of publication of articles that use the data to make scientific conclusions. 

Reviewer: I would move the comma to after but, i.e. '...data life cycle but**,** at the latest, ...'

In the EU, the 'Open Data Directive' ([Directive (EU) 2019/1024](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1561563110433&uri=CELEX:32019L1024)) states that "_Member States shall support the availability of research data by adopting national policies and relevant actions aiming at making publicly funded research data openly available (‘open access policies’), following the principle of ‘open by default’ and compatible with the FAIR principles._"

Many research funders, institutions and reputable journals/publishers now have data sharing mandates, from which you normally cannot opt out of unless there are legitimate reasons. 

* Additional reasons to share your datasets:
  - [Ten reasons to share your data.](https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data)
  - [Ask not what you can do for open data; ask what open data can do for you.](http://blogs.nature.com/naturejobs/2017/06/19/ask-not-what-you-can-do-for-open-data-ask-what-open-data-can-do-for-you/)

> Eventhough all data might not be possible to share openly because of ethical, legal, contractual, or intellectual property reasons, do strive to make data "_as open as possible, as closed as necessary_". 

Reviewer 2: ```suggestion
Even though it may not be possible to openly share all data because of ethical, legal, contractual, or intellectual property reasons, do strive to make data "_as open as possible, as closed as necessary_".
```

## What should be considered for Data Sharing?

* Take Data Sharing into consideration already at the [Planning stage](planning). Don't leave it to the last stage. 

> * Making the data as FAIR as possible will ensure that the most value can be made out of it in the future.

Reviewer 3: You might want to add a separate point about data usage licenses (FAIR principle R.1.1), since this can be critical for making shared data reusable.

Reviewer 2: We can link to `pages/your_problem/data_licence.md` for licenses.

Reviewer 4: @nsoranzo on a small note, we decided this week to rename  `data_licence.md` to `licensing.md`

Reviewer 2: > on a small note, we decided this week to rename `data_licence.md` to `licensing.md`

Fine by me!

  - The easiest way to make data FAIR is to deposit it in a publicly accessible recognised repository.
  - If there are Dicipline-specific repositories available for you data, this should be your primary choice. They will work towards a high level of FAIRness by recommending appropriate community standards for describing the data. 

>     - Some recommendations for _Dicipline-specific repositories_:

Reviewer 2: ```suggestion
    - Some recommendations for _Discipline-specific repositories_:
```
